docs/guides/getting-started/user-step-by-step.primary.in.rst

```diff
@@ -0,0 +1,128 @@
+
+      .. grid-item::
+
+         .. image:: images/acc_institution.png
+
+5. Onboarding flow - self signup pages:
+
+   - Provide your given and family name.
+   - *We do not require your legal name*; the reason you are being asked is to allow us to establish you are entitled to our data products.
+     You should supply whichever name you use for publications or are known to your colleagues as.
+   - *Examples.* If you go by your middle name, supply your middle name as the given name.
+     If you publish as *Lady Gaga*, don't sign up as *Stefani Joanne Angelina Germanotta*.
+   - Provide any email address that you can immediately access (for the confirmation in Step 6).
+   - Using your institutional email address, if you're in the US or Chile, helps our data rights verification process (Step 8) but is not mandatory.
+
+6. Onboarding flow - email confirmation:
+
+   - You will receive an email to the address you provided, from registry@cilogon.org.
+     Please look out for it in your spam folder, the subject will be "Please verify your LSST Registration".
+     Click on the link inside it to verify your email address and proceed to Step 7.
+
+7. Onboarding flow — select a username:
+
+   - Clicking on the link in your email will take you to a page where you can accept your invitation.
+   - Select a username.
+     It has to be a valid Unix username.
+   - At this point your petition for a new RSP account is complete, and it proceeds automatically to Step 8.
+
+8. Data rights verification:
+
+   - Rubin staff will assess whether you have Rubin data rights based on your name, email, and institution.
+   - This verification process can take a few days and your patience is appreciated.
+   - When data rights are confirmed, and your petition is approved, you will receive an email notification.
+   - If data rights are unclear, you will receive an email with the subject "Confirming your data rights for the Rubin Science Platform".
+     Rubin staff will help you through the data rights verification process. Please check your spam folder.
+   - After 1 month without a response, petitions will be denied, but this is not permanent.
+     Please restart the process at any time, and see Step 10 if you need assistance.
+
+9. Log in and get started.
+
+   - After receiving the email notification that your account is approved, return to the RSP at |rsp-url| and log in.
+   - It is recommended to link additional identities so that you have options for account access.
+   - Find information about the available data releases and resources for scientists, such as documentation and tutorials, at `rubinobservatory.org <https://rubinobservatory.org/for-scientists>`__.
+
+
+Need assistance?
+================
+
+If you encounter issues with this process or have questions, please post a new topic in the Support category of the `Rubin Community Forum <https://community.lsst.org/>`__, and Rubin staff will respond.
```

Comment:
Do you wan to link straight to Support, ie https://community.lsst.org/c/support/6 ?

Reply:
I usually like to link "Rubin Community Forum" straight to the landing page for consistency but I'll add the hotlink to the "Support category" here.